net/bootstrap_test.go

```diff
@@ -6,19 +6,20 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
blocking: why do this? why not just import assert through gx as assert and eliminate the githubby (another domain for you) path? my guess is your editor added the githubby path because you go got it at some point? in any case do we need both?

Reply:
yeah that was an auto-add.